src/client/MapleCharacter.java

```diff
@@ -7450,4 +7451,12 @@ public void resetBossEntries(){
         }
     }
 
+    public void setLatestEventPlacement(int latestEventPlacement) {
```

Comment:
This won't get saved when the client logs out of a character, is this what you intended to do?

Reply:
Nope it's not. I want this to be on the server so if someone crashes they can still get their event rewards. Their event placement should only be reset once they've collected their reward. I guess I would need to implement a handler for this then?